db/submission_test.go

```diff
@@ -59,3 +59,54 @@ func (this *DBTests) DBTestRemoveSubmission(test *testing.T) {
         }
     }
 }
+
+func (this *DBTests) DBTestFetchAttempts(test *testing.T) {
+    ResetForTesting();
+    defer ResetForTesting();
+
+    assignment := MustGetTestAssignment()
+
+    // User has submissions
+    studentAttempts, err := GetAttempts(assignment, "student@test.com");
+    if err != nil {
+        test.Errorf("Failed to get student attempts when should be a success: %v", err);
```

Comment:
In tests like this where you are not in a test case loop, you will probably want to use `test.Fatalf()` over `test.Errorf()`,
since the fatal one will stop the test from running anymore.